### ISSUE TITLE
Send email to the maintainer

### ISSUE THREAD

**Issue by oeway** [bioimage-io/collection-bioimage-io#468]
In the weekly meeting, @esgomezm suggested that we send an email to the maintainer with the links to the PR created, so they can keep track of it. I thought the maintainer will be tagged in the PR too, but it seems they cannot get an email from github either.

@FynnBe Maybe we can use an action to send the email together with the PR creation?

**Comment by FynnBe** [comment]
Often no maintainer is specified. If so I was assuming that github would notify the user also via e-mail (maybe depending on their github settings)

@constantinpape you have been tagged by the bot, did you get a notification?

Either way, I wouldn't be opposed to adding an optional email field for the maintainer (and author) and setup our own email notification.

**Comment by constantinpape** [comment]
> @constantinpape you have been tagged by the bot, did you get a notification?

Yes, for me this always worked.

**Comment by esgomezm** [comment]
The last time, I specified the maintainer and GitHub user and didn't receive an email about the model failing (https://github.com/bioimage-io/collection-bioimage-io/actions/runs/2663527207). It's true that in the last case it didn't even create the PR. Maybe that's the reason, but even though a contributor should be able to find out if there's any action failing or any problem with the model

**Comment by FynnBe** [comment]
ah, I think there is still a bug in the bioimage.io uploader. it os dropping the github_user fields! 
constantin uploads directly to zenodo I suppose...

**Comment by oeway** [comment]
The uploader should have been fixed a while ago. In fact, we checked over the meeting that the maintainer field in the PR has the @esgomezm listed but no email.

**Comment by FynnBe** [comment]
I suppose one needs to "watch" this repository (although you probably do that, no? @esgomezm )..
maybe adding an email field gives us more control...

**Comment by esgomezm** [comment]
I do but any new one contributing to it, may not do it. Also, the PR does not occur immediately so you need to wait until the the system refresh the content

**Comment by FynnBe** [comment]
the CI is already triggered via the bioimage.io uploader, so the delay is not too long (max 5 min). 
with a @ mention any user is notified. you do not have to watch the repo. 
![image](https://user-images.githubusercontent.com/15139589/179475037-ddfb6ed6-a100-441c-a2e6-2dbd44f57082.png)

as all the following discussion takes place on github and github useer accounts tend to be more persistent than (institute) email accounts, I wouldn't mind using github notifications only. I'm not opposed to an email option either though.

**Comment by constantinpape** [comment]
> I'm not opposed to an email option either though.

Yeah, I think having both would be best, i.e. if the maintainer provides a mail address we also send a short mail to it with the link to the relevant PR.
